tools/doc/html.js

```diff
@@ -165,10 +175,39 @@ function parseLists(input) {
 }
 
 
+// Gets a link to a Linux or BSD manpage.
+// For example, getManLink("open(2)") returns
+// '<a href="http://man7.org/linux/man-pages/man2/open.2.html">open(2)</a>'
+function getManLink(text) {
+  // It's safe to concatenate into HTML like this because `text` matches
+  var found = text.match(/([a-z]+)\(([0-9])\)/);
```

Comment:
I think man pages only go 1-8 actually

http://unix.stackexchange.com/questions/3586/what-do-the-numbers-in-a-man-page-mean

Changed it to use \d